### ISSUE TITLE
Zlib

### ISSUE THREAD

**Issue by mayday6971** [mattgrill/reservoir-docker#1]
When trying to do a configuration export, the following error is shown in the logs.

```
Exception: The extension 'zlib' couldn't be found. Please make sure your version of PHP was built with 'zlib' support. in Drupal\Core\Archiver\ArchiveTar->_error() (line 785 of /app/drupal/docroot/core/lib/Drupal/Core/Archiver/ArchiveTar.php).
```

The PHP zlib module needs to be enabled in the Dockerfile.

**Comment by mayday6971** [comment]
Added pull request.

https://github.com/mattgrill/reservoir-docker/pull/2

**Comment by mattgrill** [comment]
@mayday6971 👏  

I had actually just fixed this here, 7193b1b6b778049fada2091e62774cdfecc5a2d4, and pushed a new image to docker hub. Thanks so much for pointing this out.
